app/client/components/callCentreNumbers.tsx

```diff
@@ -20,8 +20,8 @@ export interface CallCentreNumbersProps {
 }
```

Comment:
Forgive silly question, I've noticed this in a few places, but is it necessary to pass these styles to the `css` function? I think object styles can be passed straight to the `css` prop, but I might be wrong (we're using string styles)